src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Results/_ResultsPage.cshtml

```diff
@@ -2,22 +2,26 @@
 @using static DevAdventCalendarCompetition.Resources.ViewsMessages
 
 @{
+    var weekNumber = Model.Key;
     var entries = Model.Value;
-    var prevDisabled = !Model.Value.HasPreviousPage ? "disabled" : "";
-    var nextDisabled = !Model.Value.HasNextPage ? "disabled" : "";
+    var prevDisabled = Model.Value != null && !Model.Value.HasPreviousPage ? "disabled" : "";
```

Comment:
is there a possibility to have null? why?

Reply:
because somebody can give wrong week number